SpatialGDK/Source/Public/SpatialConstants.h

```diff
@@ -11,6 +11,24 @@
 // These are not a type of key supported by TMap.
 using Worker_EntityId_Key = int64;
 
+enum ESchemaComponentType : int32
+{
+	TYPE_Invalid = -1,
```

Comment:
I feel like `TYPE_` is a bit too generic?
How about `SCHEMA_`?

Reply:
I agree that `TYPE_` is generic and will change it to `SCHEMA_` but I don't think `SCHEMA_` is the best term for this

Probably `COMPONENT_` but that's too long